monetio/sat/_modis_l2_mm.py

```diff
@@ -46,6 +52,10 @@ def read_dataset(fname, variable_dict):
             ds.attrs["quality_thresh"] = variable_dict[varname]["quality_flag"]
     hdf_close(f)
 
+    ds = ds.assign_coords({'lon': (['dim_0', 'dim_1'], longitude),
+                           'lat': (['dim_0', 'dim_1'], latitude)})
```

Comment:
please the formatter
```suggestion
    ds = ds.assign_coords(
        {
            'lon': (['dim_0', 'dim_1'], longitude),
             'lat': (['dim_0', 'dim_1'], latitude),
         }
     )
```